tests/chainer_tests/functions_tests/activation_tests/test_sigmoid.py

```diff
@@ -5,8 +5,8 @@
 import chainer
 from chainer.backends import cuda
 from chainer import functions
-from chainer import gradient_check
 from chainer import testing
+from chainer import utils
```

Comment:
Could you sort imports?

Reply:
Is this OK?
```python
from chainer import testing
from chainer.testing import attr
from chainer import utils
```

Reply:
Yes.  OpenStack Guideline recommends ordering imports by the full module paths.  https://docs.openstack.org/hacking/latest/user/hacking.html#imports